editor/src/components/canvas/canvas-strategies/strategies/reparent-helpers/reparent-strategy-parent-lookup.ts

```diff
@@ -105,6 +108,22 @@ export function getReparentTargetUnified(
   return targetParentUnderPoint
 }
 
+function findConditionalWithEmptyBranch(
+  tree: ElementPathTreeRoot,
+  metadata: ElementInstanceMetadataMap,
+): ElementPath | null {
+  for (const child of Object.values(tree)) {
+    if (isConditionalWithEmptyActiveBranch(child.path, metadata)) {
+      return child.path
+    }
+    const nestedConditional = findConditionalWithEmptyBranch(child.children, metadata)
```

Comment:
this wouldn't work if you have something like
```
div
  cond
    div
      cond
```

Reply:
sorry I missed this comment. as I wrote in chat, in this case the inner div should be under the mouse, meaning it will be walked independently because it's listed in elementsUnderPoint

Reply:
I don't think the 

div
  cond
    div
      cond

should be supported. That is going to work anyway for the inner div.

Reply:
yeah you're both right, my mistake